jobs/refresh-zl-warehouse.yml

```diff
@@ -23,6 +23,7 @@ configuration:
           - siteName: "cercalasource"
           - siteName: "mirebalais"
           - siteName: "hiv"
+          - siteName: "saint_marc_sspe"
```

Comment:
We've been trying to keep mirebalais and hiv last because they take the longest.  Would be good to move this before them so that it can be done sooner.